packages/beacon-node/src/util/sszBytes.ts

```diff
@@ -150,7 +152,7 @@ export function getAttDataBase64FromSignedAggregateAndProofSerialized(data: Uint
   }
 
   // base64 is a bit efficient than hex
-  return Buffer.from(
+  return toBuffer(
```

Comment:
maybe better to use `toBase64()` function as in electra-fork as suggested by @nflaig https://github.com/ChainSafe/lodestar/blob/6573be51534214257ef0cd5067c36753a065b3c8/packages/beacon-node/src/util/sszBytes.ts#L303